src/main/java/com/minecolonies/coremod/colony/TriggerColonyAchievements.java

```diff
@@ -0,0 +1,246 @@
+package com.minecolonies.coremod.colony;
+
+import com.minecolonies.coremod.achievements.ModAchievements;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Trigger the corresponding colony achievement.
+ */
+public class TriggerColonyAchievements
+{
+    private TriggerColonyAchievements()
+    {
+        throw new IllegalAccessError("Utility class");
+    }
+
+    private static final String TAG_LUMBERJACK_SAPLINGS   = "saplings";
+    private static final String TAG_LUMBERJACK_TREES      = "trees";
+    private static final String TAG_FISHERMAN_FISH        = "fish";
+    private static final String TAG_BUILDER_HUTS          = "huts";
+    private static final String TAG_GUARD_MOBS            = "mobs";
+    private static final String TAG_FARMER_CARROTS        = "carrots";
+    private static final String TAG_FARMER_POTATOES       = "potatoes";
+    private static final String TAG_FARMER_WHEAT          = "wheat";
+    private static final String TAG_MINER_DIAMONDS        = "diamonds";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1213)